homework-submission/week4-class16/src/components/Functionality.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { observer, inject } from "mobx-react";
+@inject("TodosStore")
+@observer
+class Functionality extends Component {
+  render() {
+    const { todo } = this.props;
+    const { TodosStore } = this.props;
+
+    return (
+      <div className="function-container">
+        <label className="check_box">
+          <input
+            type="checkbox"
+            onClick={() => TodosStore.handleCheck(todo.id)}
```

Comment:
Aha! it's checking done, and deleting, But what if this component have around 200 lines? which is possible...

Reply:
I didn't get your point here